### PR TITLE
feat(stack): add postgis stack

### PR DESCRIPTION
**Pull request by tepelbaum**

### CODE REVIEW

File: docker-compose.postgis.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  postgis_db:
+    image: postgis/postgis:16-3.4-alpine
+    restart: always
+    container_name: postgis_db
+    healthcheck:
+      test: [ "CMD-SHELL", "pg_isready" ]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+    # command: postgres -c max_connections=200 -c idle_session_timeout=150000000
```
Comment (doriankodelja-se) [inline docker-compose.postgis.yml:12]: delete ?

File: docker-compose.postgis.override.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+services:
+  postgis_db_admin:
+    ports:
+      - ${postgis_pgadmin_port:-5052}:80
+    env_file:
```
Comment (doriankodelja-se) [inline docker-compose.postgis.override.yml:5]: no need to redefine from default

File: docker-compose.postgis.yml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+      - traefik.docker.network=traefik-network
+      # Make Traefik use this domain in HTTPS
+      - traefik.http.routers.postgis-pgadmin-https.entrypoints=https
+      - traefik.http.routers.postgis-pgadmin-https.rule=Host(`${pgadmin_url}`)
+      - traefik.http.routers.postgis-pgadmin-https.tls=true
+      # Use the "le" (Let's Encrypt) resolver
+      - traefik.http.routers.postgis-pgadmin-https.tls.certresolver=le
+      # Plug middleware to redirect HTTP to HTTPS
+      - traefik.http.routers.postgis-pgadmin-http.middlewares=https-redirect
+      - traefik.http.routers.postgis-pgadmin-https.middlewares=secured
```
Comment (doriankodelja-se) [inline docker-compose.postgis.yml:43]: hopefully this won´t  have to change ever

**Review by doriankodelja-se — APPROVED** [review]
LGTM

**Comment by doriankodelja-se** [comment]
Oh yeah,one additional question: Do we need a dedicated container, could't we just have another database in the same container as the normal db ?

**Comment by tepelbaum** [comment]
> Oh yeah,one additional question: Do we need a dedicated container, could't we just have another database in the same container as the normal db ?

Unfortunately not :'(. the create extension command does not work on standard postgresql containers (at least I never managed to make it work)

File: docker-compose.postgis.override.yml (hunk 2 above)
Comment (tepelbaum) [reply docker-compose.postgis.override.yml:5]: you mean the port? You prefer to break loudly? :)

File: docker-compose.postgis.yml (hunk 1 above)
Comment (tepelbaum) [reply docker-compose.postgis.yml:12]: yep you're right

**Pull request merged**